attributes/default.rb

```diff
@@ -47,3 +47,6 @@
     omnibus['build_user_password'] = '$1$QTCj0tQy$C60hWNmo8wZo.ctvDSy9p/'
   end
 end
+
+# Chef-installed bento boxes default 7-zip here. Make sure it gets in the path.
+default['7-zip']['home']    = "#{ENV['SYSTEMDRIVE']}\\Program Files\\7-zip"
```

Comment:
@schisamo dunno, but the vagrant boxes have preinstalled 7zip, so the omnibus recipe doesn't install one. The preinstalled 7zip is in `C:\Program Files\7-zip` and the easiest way to incorporate that into omnibus was to tell omnibus it lives there. Changing it here will change where production nodes eventually install it and modify the PATH to match.